.github/workflows/e2e.yml

```diff
@@ -0,0 +1,118 @@
+name: E2E Test
+
+on:
+  pull_request:
+    branches:
+      - main
+      - "release-*"
+    paths-ignore:
+      - "docs/**"
+      - "**.md"
+      - "sec-scanners-config.yaml"
+
+env:
+  NATS_MANAGER_IMAGE: europe-docker.pkg.dev/kyma-project/prod/nats-manager:latest
+  EVENTING_MANAGER_IMAGE: europe-docker.pkg.dev/kyma-project/prod/eventing-manager:latest
```

Comment:
Should we use latest image tag? or image from latest release of eventing+nats manager?

Reply:
I was wondering the same. is there a way to always get `latest-rel`?

Reply:
`latest` will ensure the current EPP changes work with the newest changes of EM and NM. If we use a release tag, we need to update the image every time there is a new release. If we prefer a stable environment by using a release tag, I'd try and see if I can refer to these env variables from somewhere else, so we don't have to remember to bump them in the test itself.

Reply:
I used `curl` to get the latest release for the Eventing Manager. Same can be done for NATS Manager as soon as there is a new release.